specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -0,0 +1,4002 @@
+        }
+      },
+      "put": {
+        "operationId": "WorkloadNetworks_CreateDhcp",
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "WorkloadNetworks"
+        ],
+        "summary": "Create dhcp by id in a private cloud workload network.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/dhcpId"
+          },
+          {
+            "name": "workloadNetworkDhcp",
+            "description": "NSX DHCP",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/WorkloadNetworkDhcp"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/WorkloadNetworkDhcp"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkloadNetworks_CreateDhcp": {
+            "$ref": "./examples/WorkloadNetworks_CreateDhcpConfigurations.json"
+          }
+        }
+      },
+      "patch": {
+        "operationId": "WorkloadNetworks_UpdateDhcp",
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "WorkloadNetworks"
+        ],
+        "summary": "Create or update dhcp by id in a private cloud workload network.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/dhcpId"
+          },
+          {
+            "name": "workloadNetworkDhcp",
+            "description": "NSX DHCP",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/WorkloadNetworkDhcp"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
202 for long running patch

Reply:
I updated this PATCH to use a 202 and now LintDiff is giving the error:
`R2005 - LongRunningResponseStatusCode
A 'PATCH' operation 'WorkloadNetworks_UpdateDhcp' with x-ms-long-running-operation extension must have a valid terminal success status code 200 or 201.
  at specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json:1983
`
Should I change it back or is this a bug in LintDiff?